tests/unit/Owncloud/OcisPhpSdk/EducationUserTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace unit\Owncloud\OcisPhpSdk;
+
+use Owncloud\OcisPhpSdk\EducationUser;
+use Owncloud\OcisPhpSdk\Ocis;
+use PHPUnit\Framework\TestCase;
+
+class EducationUserTest extends TestCase
+{
+    /**
+     * @return void
+     */
+    public function testgetEducationUsers()
+    {
+        $ocis = $this->createMock(Ocis::class);
+        /** @phan-suppress-next-line PhanUndeclaredMethod */
+        $user = $ocis->getEducationUsers();
+        $this->assertContainsOnlyInstancesOf(EducationUser::class, $user);
```

Comment:
what happens if `$user` is an empty array?
(if something went wrong and there are exactly 0 education users)

Maybe the test will pass.

For the unit tests, we will know exactly how many education users exist, so you can also do `assertCount` to check that the array of `EducationUser` has the expected size.